samples/apps/bookinfo/src/prepare_proxy.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
Why is this being copied here? We have a version in pilot.

Reply:
I agree. we need to reuse that one.